### ISSUE TITLE
 no icon for hotspot activation 

### ISSUE THREAD

**Issue by NeoTheThird** [ubports/ubports-touch#41]
When sharing cellular data on hotspot, it works, but there's no indication on the notification bar. Must go to system settings to check if toggled.
Could be a problem if unaware that hotspot is still active, draining battery + security issues.
Reported by https://launchpad.net/~mgc.lude.

**Comment by bastos77** [comment]
Hotspot Icon has been shown when active in the last release (Version 41) before OTA-1. In OTA-1 FP2 it is again away...

**Comment by bastos77** [comment]
Now OTA-1 FP2 surprisingly it is visible again. Hotspot works like charm :-)

**Comment by NeoTheThird** [comment]
Happy to hear that @bastos77, let's close this issue.

**Comment by zubozrout** [comment]
Well, I can't see the icon on my MX4 (devel) and apparently I am not the only one: https://forums.ubports.com/topic/624/hotspot-symbol-in-status-line-missing-meizu-mx4
Is this just an MX4 issue or did it reappear?

**Comment by zubozrout** [comment]
Hmm, I've managed to get the icon appear but it did require a few reboots and toggling hotspot off and on again to make it work. When the icon showed up I even got WiFi working again after I end turn the hotspot down - https://github.com/ubports/ubports-touch/issues/109 But it didn't last long and after a few attempts I had to reboot my phone to get WiFi working. Hotspot still worked but the icon wasn't showing up in the indicator area when turned on again. And I can't get this working now so no icon and no WiFi after this for me again.
